privaterelay/templates/includes/dashboard_onboarding.html

```diff
@@ -2,28 +2,48 @@
-    <div class="flx flx-row al-cntr">
-      <span class="onboarding-num">1</span>
-      <span>{% ftlmsg 'onboarding-tip-1' %}</span>
-    </div>
-    <img class="onboarding-img img-arrow-up" src="{% static 'images/dashboard-onboarding/tip1-icon.svg' %}" aria-hidden="true" alt="">
-  </li>
-  <li class="dashboard-onboarding-list-item tip-2 flx flx-col">
-    <div class="flx flx-row al-cntr">
-      <span class="onboarding-num">2</span>
-      <span>{% ftlmsg 'onboarding-tip-2' %}</span>
-    </div>
-    <img class="onboarding-img img-input-icon" src="{% static 'images/dashboard-onboarding/input-icon.svg' %}" alt="Where possible, a button to create a new alias will appear inside a form's email input.">
-  </li>
-  <li class="dashboard-onboarding-list-item tip-3 flx flx-col">
-    <div class="flx flx-row al-cntr">
-      <span class="onboarding-num">3</span>
-      <span>{% ftlmsg 'onboarding-tip-3' %}</span>
-    </div>
-    <img class="onboarding-img img-context-menu" src="{% static 'images/dashboard-onboarding/context-menu.svg' %}" aria-hidden="true">
-  </li>
-</ol>
+<section class="c-onboarding">
+  <h2>{% ftlmsg 'onboarding-headline' %}</h2>
+  <ul class="c-onboarding-steps mzp-l-columns mzp-t-columns-three">
+    <li class="c-onboarding-step">
+      <div class="c-onboarding-step-number">1</div>
+      <p class="c-onboarding-step-copy">
+        {% ftlmsg 'onboarding-tip-1' %}
+      </p>
+      <div class="c-onboarding-step-action">
+        <form action="{% url 'emails-index' %}" class="dash-create" method="POST">
+            <input type="hidden" name="api_token" value="{{ user_profile.api_token }}">
+              <button
+                  class="mzp-c-button mzp-t-product has-icon"
+                  title="{% ftlmsg 'profile-label-generate-new-alias' %}"
+                  type="submit"
+                  value="{% ftlmsg 'profile-label-generate-new-alias' %}"
+                  data-event-label="Create New Relay Alias"
+              >
+                <span class="generate-new-relay-icon"></span>
+                <span class="generate-new-alias-text">
+                {% ftlmsg 'profile-label-generate-new-alias' %}</span>
+              </button>
+          </form>
+      </div>
+    </li>
+    <li class="c-onboarding-step">
+      <div class="c-onboarding-step-number">2</div>
+      <p class="c-onboarding-step-copy">
+        {% ftlmsg 'onboarding-tip-2' %}
+      </p>
+      <div class="c-onboarding-step-graphic">
+        <img class="" src="{% static 'images/dashboard-onboarding/onboarding-step-2.svg' %}" aria-hidden="true" alt="">
+      </div>
+    </li>
+    <li class="c-onboarding-step">
+      <div class="c-onboarding-step-number">3</div>
+      <p class="c-onboarding-step-copy">
+        {% ftlmsg 'onboarding-tip-3' %}
+      </p>
+      <div class="c-onboarding-step-graphic">
+        <img class="" src="{% static 'images/dashboard-onboarding/onboarding-step-3.svg' %}" aria-hidden="true" alt="">
```

Comment:
nit: remove empty attribute tags
```suggestion
        <img src="{% static 'images/dashboard-onboarding/onboarding-step-3.svg' %}" aria-hidden="true" alt="">
```